SKILL.tsv

```diff
@@ -4048,181 +4048,181 @@ SKILL_20150428_004047	{memo X}Summon Demon type boss{nl}Demon power +#{CaptionRa
 SKILL_20150428_004070	$Creates a magic circle in front of you that increases the DEX of allies and decreases the enemy's evasion and physical defense. Party members will receive the effect without stepping on the magic circle.
 SKILL_20150428_004071	$Gives a blessing providing additional damage to the attacks of your party members and allies.
-SKILL_20150428_004072	Additional damage +#{CaptionRatio}#{nl}Attack #{CaptionRatio2}# times{nl}Target allies #{CaptionTime}#{nl}Duration 45 seconds{nl}Use 1 Holy Powder
+SKILL_20150428_004072	$Additional Damage +#{CaptionRatio}#{nl}Number of hits: #{CaptionRatio2}#{nl}Target allies #{CaptionTime}#{nl}Duration: 45 seconds{nl}Consumes 1 Holy Powder
 SKILL_20150428_004073	{memo X}Body's moving range #{CaptionRatio}#{nl}Body's attack +#{CaptionRatio2}#
 SKILL_20150428_004074	{memo X}Reflect as much as target's Physical attack{nl}Additional damage reflected #{SkillAtkAdd}#{nl}Max. duration #{CaptionTime}# seconds{nl}Range projectile, Magic defense not possible
 SKILL_20150428_004075	{memo X}Attack #{SkillAtkAdd}#{nl}Use 2 Stamina
 SKILL_20150428_004076	{#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Attack enemy with tip of the hand to cause internal damage.
 SKILL_20150428_004077	{memo X}Attack per strike #{SkillAtkAdd}#{nl}Use #{CaptionRatio}# SP per 0.5 seconds
-SKILL_20150428_004078	Max. duration #{CaptionTime}# seconds{nl}Range projectile, Magic defense possible
-SKILL_20150428_004079	Max. purchase qty. #{CaptionRatio}#
+SKILL_20150428_004078	$Maximum Duration: #{CaptionTime}# seconds{nl}Able to defend from ranged projectiles and magic
+SKILL_20150428_004079	$Maximum Purchase Quantity: #{CaptionRatio}#
 SKILL_20150428_004080	Remove magic circle nearby{nl}Max Duration 30 seconds{nl}Target #{CaptionRatio}#
 SKILL_20150428_004081	Can open the treasure chest easily
 SKILL_20150428_004082	Movement speed slightly decreases. Receive additional damage on Lightning attacks. Give continuous damage to Fire Attribute targets.
-SKILL_20150428_004083	Quick Cast: Magic Damage
-SKILL_20150428_004084	Magic damange of next skill increases.
-SKILL_20150428_004085	SP recovery speed
-SKILL_20150428_004086	Increase SP recovery speed.
+SKILL_20150428_004083	$Quick Cast: Magic Damage
+SKILL_20150428_004084	$Increases the Magic Damage of the next skill.
+SKILL_20150428_004085	$SP Recovery Speed
+SKILL_20150428_004086	$Increases the speed of SP recovery.
 SKILL_20150428_004087	{memo X}Increase Splash.
-SKILL_20150428_004088	Recover Stamina
-SKILL_20150428_004089	Continuously recover stamina.
-SKILL_20150428_004090	Increase magic defense.
-SKILL_20150428_004091	Increase Dark Property attack.
-SKILL_20150428_004092	Red Aura
-SKILL_20150428_004093	Blue Aura
-SKILL_20150428_004094	Yellow Aura
+SKILL_20150428_004088	$Stamina Recovery
+SKILL_20150428_004089	$Continuously recovers Stamina.
+SKILL_20150428_004090	$Increase Magic Defense.
```

Comment:
You changed 4086 from "increase" to "increases", shouldn't you do that here, too?